script/seed.js

```diff
@@ -14,18 +14,40 @@ async function seed() {
   ])
 
   const orders = await Promise.all([
-    Orders.create({quantity:1, price:500, timestamp:Date.now(), status:'Created', userId: 1})
+    Orders.create({quantity:1, price:500, timestamp:Date.now(), status:'Created', userId: 1}),
+    Orders.create({quantity:3, price:3000, timestamp:Date.now(), status:'Processing', userId: 2}),
+    Orders.create({quantity:2, price:2000, timestamp:Date.now(), status:'Completed', userId: 2}),
+    Orders.create({quantity:3, price:1500, timestamp:Date.now(), status:'Cancelled', userId: 1}),
+    Orders.create({quantity:1, price:1000, timestamp:Date.now(), status:'Created', userId: 2}),
   ])
   const reviews = await Promise.all([
-    Reviews.create({content:'this is ok!', rating:3, timestamp:Date.now(), userId: 1})
+    Reviews.create({content:'this is meh!', rating:2, timestamp:Date.now(), userId: 1}),
+    Reviews.create({content:'this is ok!', rating:3, timestamp:Date.now(), userId: 2}),
+    Reviews.create({content:'this is great!', rating:4, timestamp:Date.now(), userId: 2}),
+    Reviews.create({content:'this is amazing!!', rating:5, timestamp:Date.now(), userId: 1})
   ])
   const sunglasses = await Promise.all([
-    Sunglasses.create({ name:'block',price:500,inventory:10, brand:"Rayban", color:"black", shape:"round"})
+    Sunglasses.create({ name:'stingray',price:500,inventory:5, brand:"Maui Jim", color:"black", shape:"square"}),
+    Sunglasses.create({ name:'aviator',price:500,inventory:15, brand:"Ray-Ban", color:"brown", shape:"round"}),
+    Sunglasses.create({ name:'wayfarer',price:500,inventory:7, brand:"Ray-Ban", color:"black", shape:"round"}),
+    Sunglasses.create({ name:'clubmaster',price:500,inventory:3, brand:"Ray-Ban", color:"brown", shape:"square"}),
   ])
 
-  //const [order, glasses] = await Promise.all(orders, sunglasses) 
   await orders[0].addSunglasses('1')
+  await orders[1].addSunglasses('2')
+  await orders[1].addSunglasses('3')
+  await orders[1].addSunglasses('4')
+  await orders[2].addSunglasses('1')
+  await orders[2].addSunglasses('3')
+  await orders[3].addSunglasses('3')
+  await orders[3].addSunglasses('2')
+  await orders[3].addSunglasses('1')
+  await orders[4].addSunglasses('2')
+
   await reviews[0].setSunglass('1')
```

Comment:
similar to the comment above, i think all existing reviews must belong to a pair of sunglasses